### ISSUE TITLE
Question on recommended set-up (OS, etc) for older device.

### ISSUE THREAD

**Issue by gety9** [Catfriend1/syncthing-android#873]
### Version Information
- Android Version: Android 7.0 
- Device manufacturer: Samsung
- Device model: Galaxy S7 (SM-G930T)

Thank your for great app! I've been using it on new tablet  and love it. Works like a clockwork.

This is not issue, but a question.

I want to turn my old / not used Samsung Galaxy S7 into always on device for syncthing.

Do you have any recommendations how to approach it?

Currently it has Android 7.0. To which version should i update it? Are there battery efficient modifications with minimal bloat? Or it's better to stay away from them?

Basically i need only syncthing fork on device (and basic apps - file manager, browser just in case, etc)

Thank you!

**Comment by Catfriend1** [comment]
@gety9  Thanks for reaching out. I think you're better off asking at the Syncthing forum.  Running Syncthing 24/7 is not fork-specific, so you should also get valuable feedback by Syncthing (official) app users.
